src/renderer/uia/UiaRenderer.cpp

```diff
@@ -216,6 +216,11 @@ CATCH_RETURN();
     return S_OK;
 }
 
+[[nodiscard]] HRESULT UiaEngine::PaintFrame(IRenderData* /*pData*/) noexcept

```

Comment:
`UiaEngine` does not use any of the painting method? How does it work?...

Reply:
The whole point of the `UiaEngine` is to fire a UIA Event whenever something changes on the screen. So, it doesn't really care about _where_ the screen changed or anything like that. That's why most of the extra functions don't do anything.

Reply:
Thanks @carlos-zamora for the explanation :)